index.js

```diff
@@ -0,0 +1,236 @@
+const RELEVANT_FIELDS =
+  "tld,name,capital,flags,population,region,subregion,currencies,languages,borders";
+
+const getRelevantURL = (url) => `${url}?fields=${RELEVANT_FIELDS}`;
+
+const freeChildren = (
```

Comment:
Consider changing the name to something more understandable
For example: `clearCountriesContainer`?